fuse/pathfs/pathfs.go

```diff
@@ -276,7 +276,6 @@ func (n *pathInode) GetPath() string {
 	// them, them, but since this is a hot path, we take some
 	// effort to avoid allocations.
 
-	n.pathFs.pathLock.RLock()
```

Comment:
for the deadlock, you usually also want to show the stacktrace of the routines that do hold the lock.